src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -88,11 +140,9 @@ def write_value(self, value: Union[Enum, str]):
     def value(self, value: AugmentedValue):
         return self.enum_class(value)
 
-    def descriptor(self, source: str, value: AugmentedValue) -> Descriptor:
-        choices = [e.value for e in self.enum_class]
-        return dict(
-            source=source, dtype="string", shape=[], choices=choices
-        )  # type: ignore
+    def descriptor(self, value: AugmentedValue) -> DataKey:
+        # Sometimes DBR_TYPE returns as String, must pass choices still
+        return _data_key_from_augmented_value(value, choices=self.choices)
```

Comment:
Don't you calculate these above anyway?

Reply:
The test Enum signals are returning DBR_TYPE of String and don't have a choices field :shrug: I don't know if they are misconfigured? I did try getting rid of this type same as CaArrayConverter, but had to restore this method.

Reply:
Added some docstring to the EnumConverter to explain what's going on